test/test_other.py

```diff
@@ -12269,12 +12278,16 @@ def test_standalone_export_main(self):
     # We should consider making this a warning since the `_main` export is redundant.
     self.run_process([EMCC, '-sEXPORTED_FUNCTIONS=_main', '-sSTANDALONE_WASM', test_file('core/test_hello_world.c')])
 
-  @requires_wasm_eh
   def test_standalone_wasm_exceptions(self):
     self.set_setting('STANDALONE_WASM')
     self.set_setting('WASM_BIGINT')
     self.wasm_engines = []
     self.emcc_args += ['-fwasm-exceptions']
+    self.require_wasm_eh()
+    self.do_run_in_out_file_test('core/test_exceptions.cpp', out_suffix='_caught')
+    # New Wasm EH with exnref
+    self.require_wasm_exnref()
+    self.set_setting('WASM_EXNREF')
```

Comment:
Again I wonder if that can parameterized instead?

Reply:
We can, but does this run on node 22?

Reply:
You can select specific tests to to run on node canary.  They are listed in the .circleci config.

Reply:
It looks `test-wasm64` and `test-wasm64-4gb`
https://github.com/emscripten-core/emscripten/blob/2a08958957b1a98092a305a0b3d59d84c97ec248/.circleci/config.yml#L636-L675
and some tests in `test-compat`
https://github.com/emscripten-core/emscripten/blob/2a08958957b1a98092a305a0b3d59d84c97ec248/.circleci/config.yml#L726C3-L752

are running on node canary, right?

Then I wonder, why are these tests, which has `self.require_wasm_exnref()`, passing in `test-core0` and `test-other`? `test-core0` and `test-other` don't seem to run on node canary, right? They say `ok` instead of `skipped`...
```console
test_exceptions_with_and_without_longjmp (test_core.core0) ... ok (9.40s)
test_longjmp_with_and_without_exceptions (test_core.core0) ... ok (6.01s)
test_lto_wasm_exceptions (test_other.other) ... ok (8.19s)
test_standalone_wasm_exceptions (test_other.other) ... ok (2.03s)
```
test-core0 log:
https://app.circleci.com/pipelines/github/emscripten-core/emscripten/35053/workflows/a5482f96-8732-4e8f-b7ba-bce7fb9a77ab/jobs/783071
test-other log:
https://app.circleci.com/pipelines/github/emscripten-core/emscripten/35053/workflows/a5482f96-8732-4e8f-b7ba-bce7fb9a77ab/jobs/783072

Reply:
I think that is because v8 is installed, which also satisfies the `require_wasm_exnref` requirement. 
 See https://app.circleci.com/pipelines/github/emscripten-core/emscripten/35053/workflows/a5482f96-8732-4e8f-b7ba-bce7fb9a77ab/jobs/783041

Reply:
Added `@requires_wasm_exnref` to all relevant tests I modified here, and removed `self.require_wasm_eh()` and `self.require_wasm_exnref()`. Didn't add them to the list of node-canary tests, given that they can run on v8 anyway.